src/screens/AirdropDashboard/components/CommunityGoalsListItem.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import ProgressBar from 'react-native-progress/Bar';
+
+import { icons } from 'app/assets';
+import { Image } from 'app/components';
+import { lastThresholdBeforeInfinity } from 'app/helpers/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../../loc');
+
+interface CommunityGoalsListItemProps {
+  threshold: number;
+  reward: string;
+  currentUsersQuantity: number;
+  previousThreshold: number;
+}
+
+export const CommunityGoalsListItem: FC<CommunityGoalsListItemProps> = props => {
+  const { threshold, reward, currentUsersQuantity, previousThreshold } = props;
+
+  const localUsersQuantity = currentUsersQuantity - previousThreshold;
+  const localThreshold = threshold - previousThreshold;
+  const locked = localUsersQuantity < 0;
+  const progress = locked ? 0 : localUsersQuantity / localThreshold;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.headerContainer}>
+        {locked && <Image source={icons.lockIconGrey} style={styles.lockImage} />}
+        <Text style={[styles.header, ...[locked ? [styles.lockedText] : []]]}>
+          {i18n.formatString(i18n.airdrop.community.unlockToIncrease, {
+            reward,
+          })}
+        </Text>
+      </View>
+      <ProgressBar
+        borderWidth={0}
+        unfilledColor={palette.lightGrey}
+        color={'#F7C331'}
```

Comment:
Don't you want to add the color to the palette?

Reply:
not really, it s only in one place here, I don't think it s worth adding it Not sure, can add if you prefer

Reply:
I mean, as a variable it'll be a human-friendly name there instead of hex. Up to you.